manifests/backend.pp

```diff
@@ -41,8 +37,6 @@
   Hash $config_hash = {},
   String $url_host = '127.0.0.1',
   Stdlib::Port $url_port = 8080,
-  String $username = 'admin',
-  String $password = 'P@ssw0rd!',
```

Comment:
Could we keep these if we tell people how to set it manually and change the hiera data?

Could we use Tasks to help?

Reply:
The way we use both username and password at this time is the initial bootstrap with `sensuctl configure` and that must be `admin` and `P@ssw0rd!`.  After initial bootstrap it can be changed to something else and the parameters wouldn't actually change anything since they are only used for bootstrap.  If we left the parameters it may mislead people to think they have some purpose when really they aren't yet used.  A task could be useful but would be just as easy for us to document the commands to use for sensuctl to change the password.

Reply:
Makes sense. Please add more to the docs and this will be ready for merge.

Reply:
Updated README with commands to perform the manual steps of changing password.  Let me know if more is needed.